<?xml version="1.0" encoding="UTF-8"?><testrun name="TestStudentMailMatcher.test_0_nazwa_Konta_0_Znakow_Expect_false" project="BT-Library-Java" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testcase name="test_0_nazwa_Konta_0_Znakow_Expect_false" classname="bt.algorithms.strings.TestStudentMailMatcher" time="0.0">
    <failure>junit.framework.AssertionFailedError&#13;
	at junit.framework.Assert.fail(Assert.java:55)&#13;
	at junit.framework.Assert.assertTrue(Assert.java:22)&#13;
	at junit.framework.Assert.assertFalse(Assert.java:39)&#13;
	at junit.framework.Assert.assertFalse(Assert.java:47)&#13;
	at junit.framework.TestCase.assertFalse(TestCase.java:219)&#13;
	at bt.algorithms.strings.TestStudentMailMatcher.test_0_nazwa_Konta_0_Znakow_Expect_false(TestStudentMailMatcher.java:47)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)&#13;
	at java.lang.reflect.Method.invoke(Method.java:498)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:176)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:141)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:122)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:142)&#13;
	at junit.framework.TestResult.run(TestResult.java:125)&#13;
	at junit.framework.TestCase.run(TestCase.java:129)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:252)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:247)&#13;
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:86)&#13;
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:86)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</failure>
  </testcase>
</testrun>
